Shoko.Server/Server/ShokoServer.cs

```diff
@@ -125,6 +126,10 @@ internal static void ConfigureServices(IServiceCollection services)
                 loggingBuilder.AddNLog(new ConfigurationBuilder()
                     .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true).Build());
             });
+	    services.Configure<KestrelServerOptions>(options =>
```

Comment:
A: Indentation
B: this should be in Startup.cs

Reply:
fixed in 2nd commit